src/DynamoCore/Configuration/StyleItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Configuration
+{
+    /// <summary>
+    /// This class stores the group styles added by the user
+    /// </summary>
+    public class StyleItem
+    {
+        /// This property will contain the Group Name of the stored style
+        public string GroupName { get; set; }
```

Comment:
I think you will need to move this property into GroupStyleItem as it is related to that class.